File: JOSÉ_ALEJANDRO_GÓMEZ_PAREDES_TLL5_01.py
# metodo path de la librerìa pathlib
from pathlib import Path

# creando el archivo junto con la ruta

Fl_Ruta_File = Path("C:/Users/UNEMI/Pictures/Nueva carpeta/Arcivo190523_01.txt")

# obtener ruta general absoluta
Lf_Ruta_Absoluta=Fl_Ruta_File.absolute()
#print(Lf_Ruta_Absoluta)

# verificar si una ruta existe
Lb_Existe = Fl_Ruta_File.exists()
print("¿La ruta existe?::  ", Lb_Existe)

# obtener nombre del archivo


Lf_nombre_file = Fl_Ruta_File.name
print("nombre del archivo: ",Lf_nombre_file)

#obtener la ruta padre/ raiz
print("+++++")

Lf_Ruta_raiz = Fl_Ruta_File.parent
print("El nombre del directorio raiz es: ", Lf_Ruta_raiz)

File: talleres en js/JOSÉ_ALEJANDRO_GÓMEZ_PAREDES_TLL5_01.js
const fs = require("fs");
const path = require("path");

const Fl_Ruta_File = path.join(
  "C:",
  "Users",
  "UNEMI",
  "Pictures",
  "Nueva carpeta",
  "Arcivo190523_01.txt"
);

const Lf_Ruta_Absoluta = path.resolve(Fl_Ruta_File);
console.log(Lf_Ruta_Absoluta);

const Lb_Existe = fs.existsSync(Fl_Ruta_File);
console.log("¿La ruta existe?::  ", Lb_Existe);

const Lf_nombre_file = path.basename(Fl_Ruta_File);
console.log("nombre del archivo: ", Lf_nombre_file);

console.log("+++++");

const Lf_Ruta_raiz = path.dirname(Fl_Ruta_File);
console.log("El nombre del directorio raiz es: ", Lf_Ruta_raiz);
